src/api-extension/mocks/talonone/update-customer-session-with-per-item-discount-response.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
I am making the assumption this is copied from your manual tests against a campaign with the needed setup 😅

Reply:
Yes, this will only work together with the other mocks I added. Not the easiest pattern to follow but this is how the other tests work.

Reply:
Yeah, I can imagine. But for what this is - it's great :) Thank you